src/main/java/com/alkemy/ong/controller/v1/CommentController.java

```diff
@@ -24,6 +24,13 @@
 
 	private final CommentService service;
 
+	@PutMapping("/{id}")
+	public void update(
```

Comment:
Could you pls return a 204 status (NO_CONTENT) for a successful update?

Note: use `@ResponseStatus(HttpStatus.NO_CONTENT)`